pkg/kubelet/types/constants.go

```diff
@@ -16,8 +16,22 @@ limitations under the License.
 
 package types
 
+import "runtime"
+
 const (
-	PodInfraContainerImage = "gcr.io/google_containers/pause:2.0"
+
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
+
+	DefaultPodInfraContainerImageName = "gcr.io/google_containers/pause"
```

Comment:
nit: I'd prefer this logic staying out of the kubelet package. `cmd/kubelet` seems like a better location